beacon-chain/rpc/prysm/v1alpha1/validator/proposer_bellatrix_test.go

```diff
@@ -571,17 +579,18 @@ func TestServer_GetBellatrixBeaconBlock_BuilderCase(t *testing.T) {
 		StateRoot:        make([]byte, fieldparams.RootLength),
 		ReceiptsRoot:     make([]byte, fieldparams.RootLength),
 		LogsBloom:        make([]byte, fieldparams.LogsBloomLength),
-		PrevRandao:       make([]byte, fieldparams.RootLength),
+		PrevRandao:       random,
```

Comment:
Why are these changes? Is there a new test checking the RANDAO matches in the state? Perhaps because of line 593?